### PR TITLE
Add tags variable for supported resources

### PR DESCRIPTION
**Pull request by kumoy**

### CODE REVIEW

**Comment by lorengordon** [comment]
Just closing and re-opening to trigger travis-ci...

File: main.tf [hunk 1]
```diff
@@ -75,6 +76,7 @@ resource "aws_cloudwatch_event_rule" "this" {
       }
     }
     PATTERN
+  tags          = var.tag
```
Comment (lorengordon) [inline main.tf:79]:
```suggestion
  tags          = var.tags
```

**Review by lorengordon — CHANGES_REQUESTED** [review]
@kumoy Please run `terraform fmt -recursive`, and generate the readme to include the new variable.

**Comment by kumoy** [comment]
@lorengordon made the requested changes!

**Review by lorengordon — APPROVED** [review]

**Pull request merged**